src/app/pages/OnDemandTvPage/index.test.jsx

```diff
@@ -99,14 +100,89 @@ it('should show the summary for OnDemand TV Pages', async () => {
   ).toBeInTheDocument();
 });
 
-it('should show the content unavailable message for OnDemand TV Pages', async () => {
+it('should show the video player on canonical with no live override', async () => {
+  process.env.SIMORGH_APP_ENV = 'live';
   fetch.mockResponse(JSON.stringify(pashtoPageData));
+  const { pageData } = await getInitialData('some-ondemand-tv-path');
+  const { container } = await renderPage({
+    pageData,
+    service: 'pashto',
+  });
+  const videoPlayerIframeSrc = container
+    .querySelector('iframe')
+    .getAttribute('src');
 
+  expect(videoPlayerIframeSrc).toEqual(
+    'https://polling.bbc.co.uk/ws/av-embeds/media/pashto/bbc_pashto_tv/w172xcldhhrdqgb/ps',
+  );
+});
+
+it('should show the video player on amp with no live override', async () => {
+  process.env.SIMORGH_APP_ENV = 'live';
+  fetch.mockResponse(JSON.stringify(pashtoPageData));
   const { pageData } = await getInitialData('some-ondemand-tv-path');
-  const { getByText } = await renderPage({
+  const { container } = await renderPage({
+    pageData,
+    service: 'pashto',
+    isAmp: true,
+  });
+  const videoPlayerIframeSrc = container
+    .querySelector('amp-iframe')
+    .getAttribute('src');
+
+  expect(videoPlayerIframeSrc).toEqual(
+    'https://polling.bbc.co.uk/ws/av-embeds/media/pashto/bbc_pashto_tv/w172xcldhhrdqgb/ps/amp',
+  );
+});
+
+it('should show the video player on canonical with live override', async () => {
+  process.env.SIMORGH_APP_ENV = 'test';
+  fetch.mockResponse(JSON.stringify(pashtoPageData));
+  const { pageData } = await getInitialData('some-ondemand-tv-path');
+  const { container } = await renderPage({
+    pageData,
+    service: 'pashto',
+  });
+  const videoPlayerIframeSrc = container
+    .querySelector('iframe')
+    .getAttribute('src');
+
+  expect(videoPlayerIframeSrc).toEqual(
+    'https://polling.test.bbc.co.uk/ws/av-embeds/media/pashto/bbc_pashto_tv/w172xcldhhrdqgb/ps?morph_env=live',
+  );
+});
+
+it('should show the video player on amp with live override', async () => {
+  fetch.mockResponse(JSON.stringify(pashtoPageData));
+  const { pageData } = await getInitialData('some-ondemand-tv-path');
+  const { container } = await renderPage({
+    pageData,
+    service: 'pashto',
+    isAmp: true,
+  });
+  const videoPlayerIframeSrc = container
+    .querySelector('amp-iframe')
+    .getAttribute('src');
+
+  expect(videoPlayerIframeSrc).toEqual(
+    'https://polling.test.bbc.co.uk/ws/av-embeds/media/pashto/bbc_pashto_tv/w172xcldhhrdqgb/ps/amp?morph_env=live',
+  );
+});
+
+it('should show the expired content message if episode is expired', async () => {
+  const clonedPashtoPageData = clone(pashtoPageData);
+  clonedPashtoPageData.content.blocks[0].versions = [];
+  const pashtoPageDataWithExpiredEpisode = clonedPashtoPageData;
+  fetch.mockResponse(JSON.stringify(pashtoPageDataWithExpiredEpisode));
```

Comment:
assocPath can do this directly
```suggestion
  const pageDataWithoutVersions = assocPath(['content', 'blocks', 0, 'versions'], [], pashtoPageData);
  fetch.mockResponse(JSON.stringify(pageDataWithoutVersions));
```
https://ramdajs.com/docs/#assocPath